tests/console/golang.pm

```diff
@@ -0,0 +1,52 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test the 2 newest go versions by compiling and running man_or_boy.go
+# Maintainer: Dominik Heidler <dominik@heidler.eu>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils;
+use registration;
+
+sub mob_test {
+    assert_script_run "go build /home/$username/data/man_or_boy.go";
+    assert_script_run('test $(./man_or_boy) == "-67"');
+    assert_script_run 'rm man_or_boy';
+    assert_script_run("test \$(go run /home/$username/data/man_or_boy.go) == \"-67\"");
+}
+
+sub run {
+    select_console 'root-console';
```

Comment:
Why not `$self->select_serial_terminal` ?

Reply:
updated